<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynthesiaSnifferClass</class>
 <widget class="QMainWindow" name="SynthesiaSnifferClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SynthesiaSniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="txtInfoLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Current Data </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtErrorLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentInfo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SynthesiaSniffer.qrc"/>
 </resources>
 <connections/>
</ui>
